python/link_prediction/link_prediction.py

```diff
@@ -0,0 +1,167 @@
+from email import message
+import mgp  # Python API
+import torch
+import dgl  # geometric deep learning
+from typing import Tuple, Dict
+from tests.processing_tests import test_conversion
+from numpy import int32
+from dataclasses import dataclass
+
+
```

Comment:
This file (link_prediction.py) should be on top of python module after you have finished development.